server/backend/commons.py

```
@@ -615,3 +615,23 @@ def _find_tool(tools: list | None, arguments_str: str) -> str | None:
 def _is_correct_tool(required_params: list, model_argument_list: list) -> bool:
     return set(required_params).issubset(model_argument_list)
+
+
+# some code level determinism for tool call arguments
+# - if models gives `cmd` as the key, then convert to `command`, for a bash call, as Pi expects it
+# - remove `-R` from `ls` calls, to avoid sending large amt of unwanted filenames to model
+# - Add a default timeout of 60s incase model doesnt run with a timeout of large timeouts
+def optimize_arguments(tool_name: str, model_args: dict) -> dict:
+    new_args = {("command" if k == "cmd" else k): v for k, v in model_args.items()}
+
+    if tool_name == "bash" and new_args.get("command") != None:
```

Ruff (0.15.20)

[error] Comparison to `None` should be `cond is not None`

Replace with `cond is not None`

(E711)